robots/alexander_description/README.md

```diff
@@ -0,0 +1,3 @@
+# IHMC, Boardwalk - Alexander Humanoid ('Alex' was preliminary upper body design)
+
+upstream: [](https://github.com/ihmcrobotics/ihmc-open-robotics-software/tree/develop/open-alexander/src/main/resources/models)
```

Comment:
1. Have you defined the license of this project? If so, please include the information here.
2. The link is not properly displayed. See this https://github.com/stephenmcc/example-robot-data/tree/ihmc-alexander/robots/alexander_description.